### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by maorlipchuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+oVirt Disaster Recovery
+=========
+
+The `oVirt.disaster-recovery` role responsible to manage the disaster recovery scenarios in oVirt.
+
+Requirements
+------------
+
+ * Ansible version 2.4 or higher
+ * Python SDK version 4.2.1-1 or higher
```
Comment (machacekondra) [inline README.md]: Use here just 4.2.1, it's version which is possible to install from pip.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+```yaml
+---
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  vars_files:
+     - ovirt_passwords.yml
+     - disaster_recovery_vars.yml
+  roles:
+     - {role: ovirt-ansible-disaster-recovery}
```
Comment (machacekondra) [inline README.md]: - oVirt.disaster-recovery

Comment (maorlipchuk) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (maorlipchuk) [reply README.md]: done

**Pull request merged**